var/spack/repos/builtin/packages/ocaml/package.py

```diff
@@ -12,9 +12,7 @@ class Ocaml(Package):
 
     homepage = "http://ocaml.org/"
     url      = "https://caml.inria.fr/pub/distrib/ocaml-4.06/ocaml-4.06.0.tar.gz"
-
```

Comment:
OCD: can we keep whitespaces and just add the patch?

Reply:
I agree